metrics/sinks/vpa/container_utilization.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vpa
+
+import (
+	"github.com/pkg/errors"
+	"k8s.io/heapster/metrics/core"
+	"time"
+)
+
+type utilizationSnapshotSet struct {
+	Containers []*containerUtilizationSnapshot `json:"containers"`
+}
+
+func (set *utilizationSnapshotSet) addContainer(utilizationSnapshot *containerUtilizationSnapshot) {
+	set.Containers = append(set.Containers, utilizationSnapshot)
+}
+
+type containerUtilizationSnapshot struct {
+	CreateTime     time.Time `json:"createTime"`
+	ScrapTime      time.Time `json:"scrapTime"`
+	ContainerName  string    `json:"containerName"`
+	ContainerImage string    `json:"containerImage"`
+	PodId          string    `json:"podId"`
+	// Amount of requested memory.
+	// Units: Bytes.
+	MemoryRequest int64 `json:"memoryRequest"`
```

Comment:
I guess we should probably use maps here. CPU and Memory are not the only resources we may want to use.